arm/Microsoft.Compute/disks/.parameters/min.parameters.json

```
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "sxx-az-disk-x-002"
```

In line with resource naming ref #796
```suggestion
            "value": "sxx-az-disk-min-001"
```